source/renderer/app/features/discreet-mode/ui/discreet-toggle/DiscreetToggle.js

```diff
@@ -1,22 +1,23 @@
 // @flow
 import React from 'react';
+import { observer } from 'mobx-react';
 import SVGInline from 'react-svg-inline';
+import { injectIntl } from 'react-intl';
 import classNames from 'classnames';
 import revealIcon from '../../../../assets/images/reveal-key.inline.svg';
 import hideIcon from '../../../../assets/images/hide-key.inline.svg';
+import { useDiscreetModeFeature } from '../../context';
 import styles from './DiscreetToggle.scss';
 
-type Props = {
-  className?: string,
-  isDiscreetMode?: boolean,
-  onToggle: Function,
-};
-
-const DiscreetToggle = ({
+export const DiscreetToggleButton = ({
```

Comment:
Let's change the name to `DiscreetModeToggle` (it's not a button because it's a toggle :D) and i think the word "mode" is important here because it's not a "discreet toggle" but a toggle that switches discreet _mode_ on/off